src/ui/global.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --kabisa-black: #18242b;
+  --kabisa-white: #ffffff;
+  --kabisa-green: #24b371;
+  --bp-768: 768px;
```

Comment:
**suggestion**: The value is now tied to the variable name, if for whatever reason we want to change the value, the name would not match anymore. Something like `--bp-desktop` could maybe be a better option?

Reply:
Good point, but I don't see this to be changed often. It's hard to come up with a good name since anything above 768 is also not 'desktop' or vice versa.

Reply:
Im fine with either, was kind of a nitpick :)